frontend/src/pages/CoursePage.vue

```diff
@@ -83,9 +83,7 @@ export default {
   }
   .grid-container {
     display: flex;
-    flex-direction: column;
-    align-items: center;
-    gap: 20px;
+    justify-content: space-between;
```

Comment:
.grid-container should be named "grid-course-container". When renamed, the justify-content: space-between; is going to break the entire layout of the website. Should be reverted.
![Skjermbilde 2024-04-24 214212](https://github.com/Kenzirey/Web-App-Group-15/assets/111010083/7629218c-5f78-4ab6-8d67-cdde7c47a96b)

Reply:
Forgot to mention, the grid-course-container misshap is my fault, I forgot to change the name in the css